### PR TITLE
Feed fastly

### PR DESCRIPTION
**Pull request by dantavori**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added Fastly Feed (need to test)

### CODE REVIEW

File: Packs/FeedFastly/Integrations/FeedFastly/FeedFastly.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        {
+            'indicator_type': FeedIndicatorType.IPv6CIDR,
+            'extractor': 'ipv6_addresses[].{ip:@}'
+        }
+    ]
+
+    for item in indicator_types:
+        params['indicator_type'] = item['indicator_type']
+        params['extractor'] = item['extractor']
+        feed_main(params, 'Fastly Feed', 'fastly')
```
Comment (avidan-H) [inline Packs/FeedFastly/Integrations/FeedFastly/FeedFastly.py]: We're calling `feed_main` twice? Once for each indicator type - this seems silly. There's got to be a better way to do this.

File: Packs/FeedFastly/Integrations/FeedFastly/FeedFastly.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+- defaultvalue: indicatorType
+  name: feedExpirationPolicy
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+  required: false
+  type: 17
+- defaultvalue: '20160'
```
Comment (avidan-H) [inline Packs/FeedFastly/Integrations/FeedFastly/FeedFastly.yml]:
```suggestion
- defaultvalue: '20160'
  display: ""
```

File: Packs/FeedFastly/Integrations/FeedFastly/FeedFastly.yml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- defaultvalue: indicatorType
```
Comment (avidan-H) [inline Packs/FeedFastly/Integrations/FeedFastly/FeedFastly.yml]:
```suggestion
- defaultvalue: indicatorType
  display: ""
```

File: Packs/FeedFastly/Integrations/FeedFastly/FeedFastly.yml [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Fastly Edge Cloud Platform enhances web and mobile delivery by accelerating dynamic assets and caching unpredictable and changing content. The CDN transforms and serves images faster from the edge, reducing origin traffic and saving on infrastructure and egress costs.
```
Comment (avidan-H) [inline Packs/FeedFastly/Integrations/FeedFastly/FeedFastly.yml]: This isn't so much a description of the feed as much as it seems to be a description of what Fastly, the company, presumably does. We need to consider removing/revamping this to be applicable.

File: Packs/FeedFastly/Integrations/FeedFastly/FeedFastly.yml [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+      description: limits the number of context indicators to output
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    deprecated: false
+    description: Fetches indicators from the feed.
+    execution: false
+    name: fastly-get-indicators
+  isfetch: true
```
Comment (avidan-H) [inline Packs/FeedFastly/Integrations/FeedFastly/FeedFastly.yml]:
```suggestion
  feed: true
```

File: Packs/FeedFastly/Integrations/FeedFastly/FeedFastly_description.md [hunk 6]
```diff
@@ -0,0 +1,2 @@
+### Fastly Feed
+Fastly Edge Cloud Platform enhances web and mobile delivery by accelerating dynamic assets and caching unpredictable and changing content. The CDN transforms and serves images faster from the edge, reducing origin traffic and saving on infrastructure and egress costs.
```
Comment (avidan-H) [inline Packs/FeedFastly/Integrations/FeedFastly/FeedFastly_description.md]: Same comment as I left for the duplicate description in the YAML file.

File: Packs/FeedFastly/TestPlaybooks/playbook-FeedFastly_test.yml [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+      "dimensions": {
+        "height": 735,
+        "width": 380,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs: []
+outputs: []
```
Comment (avidan-H) [inline Packs/FeedFastly/TestPlaybooks/playbook-FeedFastly_test.yml]:
```suggestion
outputs: []
fromversion: 5.5.0
```

**Comment by ronykoz** [comment]
@avidan-H + @dantavori  please take care of the description on Sunday, and merge it by end of day

**Comment by avidan-H** [comment]
@dantavori what's the status here? We should merge this so please go speak to @kirbles19 about the descriptions if you need help resolving those comments.

**Comment by avidan-H** [comment]
Furthermore, the test-playbook is failing. Please fix it asap.

**Comment by dantavori** [comment]
@avidan-H just came back from exam period today, I'll work on it today.

**Comment by dantavori** [comment]
All the parameter descriptions are standard - no technical writing review required.

File: Tests/conf.json [hunk 8]
```diff
@@ -2,10 +2,15 @@
...
+        {
+            "integrations": "Fastly Feed",
+            "playbookID": "Fastly Feed Test",
+            "fromversion": "5.5.0"
+        },
         {
             "integrations": "Malware Domain List Active IPs Feed",
             "playbookID": "Malware Domain List Active IPs Feed Test",
-            "fromversion": "5.5.0"
+            "fromversion": "5.0.0"
```
Comment (avidan-H) [inline Tests/conf.json]: Why?

Comment (dantavori) [inline Tests/conf.json]:
```suggestion
            "fromversion": "5.5.0"
```

**Pull request merged**